src/plugins/experimental-discord/fetcher.js

```diff
@@ -105,16 +105,22 @@ export class Fetcher implements DiscordApi {
   async members(
     guild: Model.Snowflake
   ): Promise<$ReadOnlyArray<Model.GuildMember>> {
-    // TODO: hack, should have pagination.
-    const members = await this._fetchJson(
-      `/guilds/${guild}/members?limit=1000`
-    );
-    if (members.length === 1000) {
-      throw new Error(
-        "TODO: getting 1000 members, needs to implement pagination"
+    const limit = 1000;
+    let doneLoading = false;
+    let allMembers = [];
+    let after = "0";
+    while (!doneLoading) {
+      const newMembers = await this._fetchJson(
+        `/guilds/${guild}/members?after=${after}&limit=${limit}`
       );
+      if (newMembers.length < limit) {
+        doneLoading = true;
+      } else {
+        after = newMembers[newMembers.length - 1];
```

Comment:
after needs to be a snowflake. as written, i believe it will be an object. surprised flow isn't complaining. suggest `after = newMembers[newMembers.length - 1].user.id`

Reply:
yes, this was revealed once tested on the 1hive instance. I did not know how to run localversions of packages before, but now that I know this will help me with testing in the future.

Reply:
Flow doesn't detect this because newMembers are detected as an any type.

Reply:
I have updated a few of the types for flow for changes on this branch.